ts/DataGrid/DataGridDefaults.ts

```diff
@@ -32,7 +32,7 @@ import type DataGridOptions from './DataGridOptions';
 /** @internal */
 const DataGridDefaults = {
     /** @internal */
-    cellHeight: 25,
+    cellHeight: 49,
```

Comment:
As I see in the design only the header row should have a more significant height.
![image](https://github.com/highcharts/highcharts/assets/57004120/a1d2f349-1595-49ef-8f24-d4083ebc9dcc)

Also, could you explain why 49?

Reply:
49 is applied only for header's cells.

That works from the design of data-grid, so I did not refactor the core of it.